jwst/resample/tests/test_resample_step.py

```diff
@@ -490,6 +490,72 @@ def test_pixel_scale_ratio_spec_miri(miri_cal, ratio, units):
     result3.close()
 
 
+@pytest.mark.parametrize("units", ["MJy", "MJy/sr"])
+@pytest.mark.parametrize("ratio", [0.7, 1.0, 1.3])
+def test_pixel_scale_ratio_spec_miri_pair(miri_rate_pair, ratio, units):
+    im1, im2 = miri_rate_pair
+    _set_photom_kwd(im1)
+    _set_photom_kwd(im2)
+    im1.meta.bunit_data = units
+    im2.meta.bunit_data = units
+    im1.meta.filename = 'file1.fits'
+    im2.meta.filename = 'file2.fits'
+    im1.data += 1.0
+    im2.data += 1.0
+
+    # Make an input pixel scale equivalent to the specified ratio
+    input_scale = compute_spectral_pixel_scale(im1.meta.wcs, disp_axis=2)
+    pscale = 3600.0 * input_scale / ratio
+
+    result1 = ResampleSpecStep.call([im1, im2])
+    result2 = ResampleSpecStep.call([im1, im2], pixel_scale_ratio=ratio)
+    result3 = ResampleSpecStep.call([im1, im2], pixel_scale=pscale)
+
+    # pixel_scale and pixel_scale_ratio should be equivalent
+    nn = np.isnan(result2.data) | np.isnan(result3.data)
```

Comment:
Not quite - they are close, but not necessarily exactly identical at the edges because of floating point precision.  For this test, when pixel ratio is 0.7, there are 2 pixels that end up NaN when pixel_scale is specified that are not NaN when the equivalent pixel_scale_ratio is specified.

Reply:
sounds good, as long as it's just edge effects